src/admin-views/settings/emails/email-template/body/footer.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+
+if ( empty( $footer_content ) && ! tribe_is_truthy( $footer_credit ) ) {
+	return;
+}
+?>
+<tr>
+	<td style="padding:0px 20px 10px 20px;border-top:1px solid #efefef;background:<?php echo $header_bg_color; ?>;">
+		<table role="presentation" style="width:100%;border-collapse:collapse;border:0;border-spacing:0;">
+			<?php if ( ! empty( $footer_content ) ) : ?>
```

Comment:
We can possibly move these atoms to its own templates so that we have early bails on the smaller pieces.